verifier/check_known_issues.py

```diff
@@ -37,6 +38,8 @@
 NBSP = '\u202f'
 SP = '\u0020'
 
+floating_point_all_zero = re.compile(r"\.[^1-9]+")
```

Comment:
We only care if the last digit after the decimal is zero. I think a better regex would be:

```suggestion
floating_point_all_zero = re.compile(r"\.\d*0$")
```